### PR TITLE
Feature/6u6g2g integrate with sanity

### PR DESCRIPTION
**Pull request by floraCodes**
In this branch:

- Added integration to work with Sanity CMS
- Fix Sanity studio deploy

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,24 +3,30 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
+    "dev": "concurrently \"next\" \"cd sanity && sanity start\"",
+    "build": "echo 'Building Sanity' && cd sanity && sanity install && sanity build public -y && cd .. && next build",
```
Comment (eltuna9) [inline package.json:7]: 🤯 well done!

File: sanity/dist/index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
+<!doctype html><html><head><meta charSet="utf-8"/><title>Techo – Sanity</title><meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"/><meta name="robots" content="noindex"/><style>html {background-color: #e4e8ed;}</style><link rel="stylesheet" href="/sanity/static/css/main.css?97523b31dae5df8d781f"/><link rel="subresource" href="/sanity/static/js/vendor.bundle.js?75a1b19ec1d6ce4d4b6c"/><link rel="subresource" href="/sanity/static/js/app.bundle.js?97523b31dae5df8d781f"/><link rel="icon" href="/sanity/static/favicon.ico"/></head><body id="sanityBody"><div id="sanity"><div class="sanity-app-loading-screen"><style type="text/css">
```
Comment (eltuna9) [inline sanity/dist/index.html:1]: Im pretty sure we could remove the /dist folder as surely it gets generated when we execute `sanity build` in vercel. 

Their example app has it gitignored

https://github.com/sanity-io/sanity-template-nextjs-ecommerce/tree/a078522b969b48e34a9327918c71eeadb3991e94/studio

File: public/sanity/static/.gitkeep [hunk 3]
```diff
@@ -0,0 +1 @@
+Files placed here will be served by the Sanity server under the `/static`-prefix
```
Comment (eltuna9) [inline public/sanity/static/.gitkeep:1]: we could also gitignore public/sanity

**Comment by eltuna9** [comment]
Some steps I had to go through to run the project (Windows)

Install lerna globally (postinstall script uses it)
`npm install --g lerna`

Install sanity/cli globally
`npm install --g @sanity/cli`

run npm install inside the sanity folder
`cd sanity && npm install`

File: package.json (hunk 1 above)
Comment (floraCodes) [reply package.json:7]: 👯

File: sanity/dist/index.html (hunk 2 above)
Comment (floraCodes) [reply sanity/dist/index.html:1]: Dale!

File: public/sanity/static/.gitkeep (hunk 3 above)
Comment (floraCodes) [reply public/sanity/static/.gitkeep:1]: gotcha

**Review by eltuna9 — APPROVED** [review]
This is massive!!

Well done 🚀

**Pull request merged**